scan/lib/scan/options.rb

```diff
@@ -151,6 +151,11 @@ def self.available_options
                                      env_name: "SCAN_FORMATTER",
                                      description: "A custom xcpretty formatter to use",
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :xcpretty_format,
+                                     env_name: "XCPRETTY-FORMAT",
```

Comment:
This should use an underscore instead of a dash. But can we also change this to `XCPRETTY_ARGS`?